dataverse-webapp/src/main/webapp/createEditDataverse.xhtml

```diff
@@ -226,13 +215,13 @@
 
                         <div class="row">
                             <div class="col-sm-12 form-group">
-                                <label class="col-sm-3 control-label">#{bundle['dataverse.metadataElements']}</label>
-                                <div class="col-sm-9">
+                                <label class="col-md-3 control-label">#{bundle['dataverse.metadataElements']}</label>
```

Comment:
It would be nice if this label would be aligned the same as fields above. Currently it have deeper left margin (same for section "Browse/Search Facets" and "Messages and banners")

Reply:
Done.